example-config.yml

```diff
@@ -414,6 +414,7 @@ itinerary:
 #       BUS: # Any OTP mode
 #         color: 00FF00 # HEX color without the # (like GTFS)
 #         textColor: FFFFFF # HEX color without the # (like GTFS)
+#     routeIcons: false # If set to false, no mode icons in the route viewer
```

Comment:
Please add this new attribute to the corresponding type in `config-types.ts`.

Reply:
Good catch thanks!